src/static/js/fantom_based.js

```diff
@@ -0,0 +1,250 @@
+  const startTime = await bShareRewardPoolContract.poolStartTime();
+  const currentTime = Date.now() / 1000
+
+  const ACROPOLIS_ADDR = "0xE5009Dd5912a68B0D7C6F874cD0b4492C9F0e5cD";
+  const ORACLE_ADDR = "0x8AA7e17f96647E08a54880521b2f3e7F99aa11ca";
+  const rewardTokenAddress = "0x8D7d3409881b51466B483B11Ea1B8A03cdEd89ae"; //BASED
+  const lptAddress = "0x6F607443DC307DCBe570D0ecFf79d65838630B56"; //BSHARE-FTM-LP
+  const stakeTicker = "BSHARE";
+  const rewardTicker = "BASED";
+  const epochsPerDay = 4;
+  const maxSupplyIncrease = await getExpansion(App);
+  const decimals = 18;
+  const ratio = 1.0;
+  const targetMantissa = 12;
+
+  let bShareRewardPool = await loadRewardPoolContract(App, tokens, prices, bShareRewardPoolContract, BSHARE_REWARD_POOL_ADDR, BSHARE_REWARD_POOL_ABI, "BSHARE",
+    "bshare", "pendingShare", 4, startTime, currentTime);
+
+  _print('-------------------------------------------------')
+  _print('')
+
+  let acropolis = await loadAcropolis(App, prices, ACROPOLIS_ADDR, ORACLE_ADDR, lptAddress, rewardTokenAddress, stakeTicker,
+    rewardTicker, epochsPerDay, maxSupplyIncrease, decimals, ratio, targetMantissa);
+
+
+  _print_bold(`Total Staked: $${formatMoney(bShareRewardPool.totalStaked + acropolis.staked_tvl)}`);
+
+  hideLoading();
+}
+
+async function loadRewardPoolContract(
+  App,
+  tokens,
+  prices,
+  contract,
+  contractAddress,
+  contractAbi,
+  rewardTokenTicker,
+  rewardTokenFunction,
+  pendingRewardsFunction,
+  poolCount,
+  startTime,
+  currentTime
+) {
+  const poolContract = contract ?? new ethers.Contract(contractAddress, contractAbi)
+
+  _print(`<a href='https://ftmscan.com/address/${poolContract.address}' target='_blank'>Marketplace Contract</a>`);
+  _print("");
+
+  const totalAllocPoints = await poolContract.totalAllocPoint()
+
+  const rewardTokenAddress = await poolContract.callStatic[rewardTokenFunction]()
+
+  const rewardToken = await getFantomToken(App, rewardTokenAddress, contractAddress)
+
+  let rewardsPerWeek = 0;
+  if (currentTime < startTime) {
+    _print(`Rewards has not started yet\n`);
+  } else {
+    rewardsPerWeek = await getTokenRewardPerSecond(poolContract) / 10 ** rewardToken.decimals * 604800;
+  }
+
+  const poolInfos = await Promise.all([...Array(poolCount).keys()].map(async (x) =>
+    await getBasedRewardPoolInfo(App, poolContract, contractAddress, x, pendingRewardsFunction)));
+
+  var tokenAddresses = [].concat.apply([], poolInfos.filter(x => x.poolToken).map(x => x.poolToken.tokens))
+
+  await Promise.all(tokenAddresses.map(async (address) => {
+    tokens[address] = await getFantomToken(App, address, contractAddress);
+  }))
+
+  const poolPrices = poolInfos.map(pInfo => pInfo.poolToken ? getPoolPrices(tokens, prices, pInfo.poolToken, "fantom") : undefined);
+
+  let aprs = [];
+  for (i = 0; i < poolCount; i++) {
+    if (i === 2) continue; // Skip team pool (made for dev allocation)
```

Comment:
Users are aware of this pool and is shown in the website here: https://basedfinance.io/marketplace
Not showing on vfat to avoid any confusion.